### PR TITLE
[TensorDec] Add functions regarding image labeling in tensor decoder

### PR DESCRIPTION
**Pull request by jinhyuck-park**
Add functions regarding image labeling
in tensor decoder

Signed-off-by: jinhyuck-park <jinhyuck83.park@samsung.com>

**Self evaluation:**
1. Build test: [*]Passed [ ]Failed [ ]Skipped
2. Run test: [ ]Passed [ ]Failed [* ]Skipped

### CODE REVIEW

File: gst/tensor_decoder/tensordec.c [hunk 1]
```diff
@@ -688,6 +688,56 @@ gst_tensordec_copy_buffer (GstTensorDec * self,
...
+      self->tensordec_image_label.labeling_info.total_labels);
+
+  for (i = 0; i < len; i++) {
+    if (scores[i] > 0 && scores[i] > max_score) {
+      index = i;
+      max_score = scores[i];
+    }
+  }
+
+  self->tensordec_image_label.new_label_index = index;
```
Comment (myungjoo) [inline gst/tensor_decoder/tensordec.c]: Why don't you return the index with the max-score?
Why do we have such values in the class (self)? It's a volatile value (effective for only one frame) that we do not need to save.

File: gst/tensor_decoder/tensordec.h [hunk 2]
```diff
@@ -66,6 +66,7 @@ typedef struct
 {
   gint current_label_index; /**< current label index */
   gint new_label_index; /**< new label index */
+  gchar *image_label;       /**< image label that will set to outbuf*/
```
Comment (myungjoo) [inline gst/tensor_decoder/tensordec.h]: The label value changes for every frame. This is a volatile value that doesn't need to be in the class.

File: gst/tensor_decoder/tensordec.c [hunk 3]
```diff
@@ -717,7 +767,8 @@ gst_tensordec_get_label (GstTensorDec * self,
       gst_memory_unmap (mem, &info);
     }
   }
-
+  self->tensordec_image_label.image_label = gst_get_image_label (self);
```
Comment (myungjoo) [inline gst/tensor_decoder/tensordec.c]: ```for (i = 0; i < num_mems; i++)```: #586 Very strange. If it's multi-tensor, only one of them will be valid for image-labels.

File: gst/tensor_decoder/tensordec.c (hunk 1 above)
Comment (jinhyuck-park) [reply gst/tensor_decoder/tensordec.c]: I will change image labeling data structure as not using values in class

File: gst/tensor_decoder/tensordec.h (hunk 2 above)
Comment (jinhyuck-park) [reply gst/tensor_decoder/tensordec.h]: I will change it as does not use volatile value in class

File: gst/tensor_decoder/tensordec.c (hunk 3 above)
Comment (jinhyuck-park) [reply gst/tensor_decoder/tensordec.c]: I will consider it for next step

**Pull request merged**